website/docs/language/settings/backends/remote.html.md

```diff
@@ -8,17 +8,22 @@ description: |-
 
 # remote
 
-**Kind: Enhanced**
+-> **Note:** With the release of Terraform v1.1.0, we recommend using the Terraform Cloud's built-in
+[`cloud` integration](/docs/language/settings/terraform-cloud.html) instead of this backend.
+The `cloud` option includes an improved user experience and more features.
 
--> **Note:** We recommend using Terraform v0.11.13 or newer with this
-backend. This backend requires either a Terraform Cloud account on
-[app.terraform.io](https://app.terraform.io) or a Terraform Enterprise instance
-(version v201809-1 or newer).
+-> **Note:** This backend is unique among all other Terraform backends in that it has the ability to
+execute operations for Terraform Cloud's [CLI-driven run workflow](/docs/cloud/run/cli.html), rather
+than only store state snapshots. (The documentation used to refer to this as "enhanced" backend
+behavior, but it's simpler to describe it as a quirk of the `remote` backend.)
+
+-> **Note:** The remote backend requires Terraform v0.11.13 or newer, and requires either a
+Terraform Cloud account on [app.terraform.io](https://app.terraform.io) or a Terraform Enterprise
+instance (version v201809-1 or newer).
```

Comment:
We're very inconsistent in how we do this, but the language I've seen most often is "later" rather than "newer."
```suggestion
-> **Note:** The remote backend requires Terraform v0.11.13 or later, and requires either a
Terraform Cloud account on [app.terraform.io](https://app.terraform.io) or a Terraform Enterprise
instance (version v201809-1 or later).
```